.projenrc.js

```diff
@@ -14,11 +16,17 @@ const project = new TypeScriptProject({
     'By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.',
   ],
 
-  releaseToNpm: true,
+  // docgen: true,
   codeCov: true,
   defaultReleaseBranch: 'main',
   repository: 'github:myhelix/multi-convention-namer',
 
+  // JSII options
+  compat: true,
+  catalog: true,
+  releaseToNpm: true,
+  publishToGo: { moduleName: 'multiConventionNamer' },
```

Comment:
oooo is this actually publishing to a git repo?

Reply:
@corrjo not yet. This is my first pass at figuring it out.